Documentation/operations/upgrade.rst

```diff
@@ -330,6 +330,11 @@ Helm Options
   were deprecated in Cilium 1.14 in favor of ``tls.ca.cert`` and ``tls.ca.key`` respectively,
   and have been removed. The ```clustermesh-apiserver-ca-cert`` secret is no longer generated.
 
+Changed Metrics
+~~~~~~~~~~~~~~~
+
+* ``kvstore_operations_duration_seconds`` does not include client-side rate-limiting latency anymore.
```

Comment:
```suggestion
* ``cilium_kvstore_operations_duration_seconds`` does not include client-side rate-limiting latency anymore.
```
I would use full metric names in documentation, to avoid confusion (unfortunately it's not consistent in the upgrade notes for earlier releases).

Reply:
I've added also other variants of this metric that are affected + information on what metric to use to observe client-side rate-limiting now.